src/platform-includes/source-context/java.mdx

```diff
@@ -277,6 +331,18 @@ Next, upload that source bundle to Sentry:
 sentry-cli debug-files upload --type jvm path/to/bundle
 ```
 
+<PlatformSection supported={["android"]}>
+
+### Uploading including the Native Source Bundle
+
+Include the native source bundle with `--include-sources` and upload it to Sentry:
+
+```
+sentry-cli debug-files upload --include-sources /path/to/files --type jvm path/to/bundle
+```
+
+</PlatformSection>
```

Comment:
I'm not an expert but I just tried to upload the native symbols + sources for our Android demo app. It works a bit easier for the NDK, there's no need to create a source bundle - just adding `--include-sources` does the trick.

How about the following? (GH suggestions is not working due to code blocks):

### Uploading Native Symbols and Source Code

Upload the native symbols manually using the `debug-files upload` command, here's a typical example for Android:

```
sentry-cli debug-files upload app/build/intermediates/merged_native_libs/
```

In order to upload the native source files as well, you can use the `--include-sources` flag. Using this flag sentry-cli will automatically generate source bundles on the fly during the upload.

```
sentry-cli debug-files upload app/build/intermediates/merged_native_libs/ --include-sources
```

Reply:
that's how it works yes 👍 sentry-cli will walk the file tree on itself and search for relevant sources when `--include-sources` is supplied.